### ISSUE TITLE
Help Wanted !! Realtime Logs

### ISSUE THREAD

**Issue by mmv-dev** [pocketbase/pocketbase#1800]
Can you explain me how this realtime logs work. I am new to this realtime think. Does this logs harm the performance or is this ok. I am getting logs alot and i am little scared, did i implement the realtime wrong or other. I am using react and nextjs for my electron application. I created a separate js file and implemented all the functions related to the pocketbase in it and calling the functions when needed. Even subscriptions are also there in that file. Not part of the component or anything. 

These logs are very frequent. 

Nice to have, show what record is being used in that particular realtime request in log. Export logs.

![image](https://user-images.githubusercontent.com/67296473/217125006-49f594d8-4cd5-4578-97e7-f98cefcb59da.png)

**Comment by ganigeorgiev** [comment]
The logs indicates access requests to the API endpoints.
It is the same type of log when you make requests to list a collection records for example.

The realtime logs in particular are not very useful at the moment because the connection is persistent and you'll get the `GET /api/realtime` log only after the connection was closed. This eventually will be made more useful in the future with the generalized logs (_it is already in the [roadmap](https://github.com/orgs/pocketbase/projects/2)_).

You can disable the logs completely from the "Admin UI > Settings > Application > Logs max days retention" and set it to 0.
